lib/util.js

```diff
@@ -276,14 +285,44 @@ function formatValue(ctx, value, recurseTimes) {
     }
   }
 
-  var base = '', array = false, braces = ['{', '}'];
+  var base = '', empty, braces, formatter;
```

Comment:
Preferably assign `empty = false` and use strict checks below (i.e. `empty = empty === true && keys.length === 0` and `if (empty === true) {`.)

It's a minor thing but V8 emits marginally tighter code for strict boolean comparisons.